packages/yew/Cargo.toml

```diff
@@ -34,6 +34,7 @@ serde = { version = "1", features = ["derive"] }
 tokio = { version = "1.19", features = ["sync"] }
 tokio-stream = { version = "0.1.9", features = ["sync"] }
 tracing = "0.1.36"
+html_parser = { version = "0.6", optional = true }
```

Comment:
Do we need this, or can we just use the underlying browser to parse the html or is that not portable enough?

Reply:
There's no browser on the server to parse the HTML. This is only activated when ssr feature is enabled

Reply:
But can't we literally emit the string that's getting passed in into the output stream on the server? At least that's preserving the user input to the best of our abilities. After all, it's `from_raw`, so the less processing that's happening, the better?

Reply:
I redid the entire thing and now it does return the input string as-is, except in one case where there is more than one top level node. In order to check that, we need to parse the HTML.

That is required because a when `BNode::Raw` reconciles, it must return _one_ `NodeRef`. If there's multiple top-level nodes, they must be wrapped in a parent element. That behavior needs to be replicated on the server as well

Reply:
I think it needs to return a `NodeRef` to the first element, or the passed in `next_sibling` if there are none. The additional difficulty is then removing all nodes again when unmounting but multiple children should work, there is no inherent limitiation, I think. When in doubt, check the `BList` implementation?

Reply:
I made that change, there's no `html_parser` dependency anymore